test/regress/cli/CMakeLists.txt

```diff
@@ -1758,6 +1758,7 @@ set(regress_0_tests
   regress0/seq/err1.smt2
   regress0/seq/intseq.smt2
   regress0/seq/intseq_dt.smt2
+  regress0/seq/issue11924-semi-eval-unk.smt2
```

Comment:
Does this fix trigger only in the presence of quantifiers? If not, maybe it would be good to have a simpler quantifier-free test for this?

Reply:
At the moment, the only way I know how to trigger the bug is with quantifiers, since the body of quantified formulas are not preprocessed but evaluated in the model (which is what triggers the failure).

Notice that the quantifier in this example is completely meaningless: `f` does not occur in its body, but it does impact how it is evaluated in the model.